### PR TITLE
Change the attribute name to enable the no duplicate methods rule. 

### PR DESCRIPTION
**Pull request by vilyapilya**
Rename the attribute
Change the rules for mutable strings.
Remove autocorrection from circleci

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -130,7 +130,7 @@ Style/IfUnlessModifier:
   Enabled: false
 
 Lint/DuplicateMethods:
-  Enabled: false
+  Enabled: true
```
Comment (dchai76) [inline .rubocop.yml]: 👍

File: .rubocop.yml [hunk 2]
```diff
@@ -140,3 +140,12 @@ Style/NegatedIf:
...
   Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: true
+
+Style/RedundantFreeze:
+  Enabled: true
+
+Style/MutableConstant:
+  Enabled: false
```
Comment (dchai76) [inline .rubocop.yml]: Remove this - this should be enabled. The error this is generating is legit - see #17

File: lib/logdna.rb [hunk 3]
```diff
@@ -15,11 +15,11 @@ class Ruby < ::Logger
     # uncomment line below and line 3 to enforce singleton
     # include Singleton
     Logger::TRACE = 5
-    attr_accessor :level, :app, :env, :meta
+    attr_accessor :log_level, :app, :env, :meta
```
Comment (dchai76) [inline lib/logdna.rb]: Let's just remove `log_level` from public access altogether; i.e. remove it from this list.

File: .rubocop.yml (hunk 2 above)
Comment (vilyapilya) [reply .rubocop.yml]: As I understand this rule is for the .freeze. But if you add a string like # frozen_string_literal: true .freeze becomes redundant.

Comment (dchai76) [reply .rubocop.yml]: Again, read my comments on #17. frozen_string_literal only freezes strings. Not other objects, including objects that contain strings. This style (properly) captures all of those - we should care about more than just strings.

**Comment by smusali** [comment]
in the end, I will use the same `.rubocop.yml` for `rails` too after getting the updates merged in here

File: .rubocop.yml (hunk 2 above)
Comment (vilyapilya) [reply .rubocop.yml]: Gotcha.

**Comment by vilyapilya** [comment]
@smusali hold on with the rails changes. I am going to make a deeper research on the reasons for having a separate Rails lib in the form we have and the reasons for having it at all. We will probably get rid of it.

**Comment by smusali** [comment]
> @smusali hold on with the rails changes. I am going to make a deeper research on the reasons for having a separate Rails lib in the form we have and the reasons for having it at all. We will probably get rid of it.

@vilyapilya, not touching the codebase - just adding the same `.rubocop.yml` - we can get rid of that later and there is no harm in updating `.rubocop.yml` there

**Comment by vilyapilya** [comment]
@smusali  I mean that they work might be redundant if we decide to get rid of the rails  library.

File: .rubocop.yml (hunk 2 above)
Comment (dchai76) [reply .rubocop.yml]: The other changes look good, I think we just want to remove this now.

**Pull request merged**